core/domain/html_cleaner.py

```diff
@@ -18,9 +18,12 @@
 
 import logging
 import urlparse
+from HTMLParser import HTMLParser
```

Comment:
import HTMLParser (and then, in code, refer to HTMLParser.HTMLParser)

(and, please alphabetize: HTMLParser then json then logging then urlparse)